src/Lykke.Service.KucoinAdapter.Services/ShutdownManager.cs

```diff
@@ -16,9 +16,10 @@ public class ShutdownManager : IShutdownManager
         private readonly ILog _log;
         private readonly List<IStopable> _items = new List<IStopable>();
 
-        public ShutdownManager(ILog log)
+        public ShutdownManager(ILog log, OrderbookPublishingService publishingService)
         {
             _log = log;
+            Register(publishingService);
```

Comment:
It's better to do the opposite - register OrderbookPublishingService from it's c-tor in ShutdownManager